Dockerfile

```diff
@@ -1,8 +1,38 @@
-FROM vmearl/sideboard:latest
+FROM ghcr.io/magfest/sideboard:main
 MAINTAINER RAMS Project "code@magfest.org"
 LABEL version.rams-core ="0.1"
 
-# add our code
-COPY . plugins/uber/
-# go ahead and install base dependencies
+# install ghostscript and gettext-base
+RUN apt-get update && apt-get install -y ghostscript gettext-base vim && rm -rf /var/lib/apt/lists/*
+
+ADD requirements*.txt plugins/uber/
+
 RUN /app/env/bin/paver install_deps
+RUN /app/env/bin/python3 -m pip install SQLAlchemy==1.3.0
+
+ADD . plugins/uber/
+
+ADD uber-development.ini.template ./uber-development.ini.template
+ADD sideboard-development.ini.template ./sideboard-development.ini.template
+ADD uber-wrapper.sh /usr/local/bin/
+
+# These are just semi-reasonable defaults. Use either -e or --env-file to set what you need
+# I.e.:
+# docker run -it -e HOST=192.168.0.10 -e PORT=80 ghcr.io/magfest/sideboard:main
+# or
+# echo "HOST=192.168.0.10" > uberenv
+# docker run -it --env-file uberenv ghcr.io/magfest/sideboard:main
```

Comment:
Shouldn't it be ghcr.io/magfest/ubersystem:main here? 😅

Reply:
Whoops.